### PR TITLE
Nerfs Curse of the Cluwne, Nerfs Cluwnes themselves

### PR DESCRIPTION
**Pull request by TheOneAndOnlyCreeperJoe**
Better version of #2230 

Cluwnes have a 33% chance of dropping an item instead of throwing it.
Cluwnes have a 33% chance of dropping their current item if they attack someone with it.
Cluwnes have a 33% chance of dropping a spray bottle when you spray with it.
Curse of the Cluwne can no longer be upgraded.
Gave Curse of the Cluwne some QoL changes that stops you from wasting your cooldowns on incompatible mobs."

aka if you cast it on poly by accident it will just reset the cooldown and return.
nerf forces Curse of the Cluwne to be used more selectively rather than being thrown onto everyone you can find.

### CODE REVIEW

File: code/datums/spells/barnyard.dm [hunk 1]
```diff
@@ -54,32 +54,40 @@
...
+	var/mob/living/carbon/human/target = targets[1]
 
 	if(!(target.type in compatible_mobs))
 		user << "<span class='notice'>You are unable to curse [target]!</span>"
+		castfail()
 		return
 
 	if(!(target in oview(range)))
 		user << "<span class='notice'>They are too far away!</span>"
+		castfail()
```
Comment (JohnGinnane) [inline code/datums/spells/barnyard.dm:90]: Could this be shortened to:

`if(!(target in oview(range)) || !(target.type in compatible_mobs)))`

Comment (TheOneAndOnlyCreeperJoe) [inline code/datums/spells/barnyard.dm:90]: each gives an unique error message so you know what went wrong, which may be handy to know.

Comment (JohnGinnane) [inline code/datums/spells/barnyard.dm:90]: Oh right sorry, I hadn't noticed that!

**Comment by JohnGinnane** [comment]
Code looks good!

**Comment by trifornt-tinfornt** [comment]
Holy shit Creeper Joe, we just talked about this.
Making it so you can't upgrade it is already interesting enough, but increasing the cooldown to 100 will make literally _nobody_ use it over Disintegrate.
We get it, you really want to make Cluwnes be like their goon counterparts (despite the fact that goon is kind of a different bag than Hippie), but this is overkill in a nerf for the spell.

What's the point of a gimmick item if it's incredibly slow to recharge, can't be upgraded so you can't even speed it up and ultimately just makes ONE player's life harder when you could get Disintegrate, which rewards you for using the spell by instantly killing someone and making them explode into a shower of gore?

**Comment by TheOneAndOnlyCreeperJoe** [comment]
You're almost entirely incapable of fighting back at this point, if you drop your weapon every third attack on average then you are almost incapable of dealing any damage outside of punching. The current state of the spell is that you use it on anyone you come across like you would with disintegrate. The reason why I nerfed it this hard is to incentive saving the spell for dangerous targets, such as the HoS/Captain/Powergamers, rather then using it on the assistant you just randomly found wandering the halls. If you get Cluwned as either of these 3, your days are over. In addition, it still has it's 3 tile range, so you can use it while stunned on anyone that tries to melee you (won't save you from a lynchmob though)

90-100 seconds (because the numbers on the cooldown fluctuate depending on how BYOND feels like today) means that in a 30 minute round you can still cluwne well around ~18 people as opposed to the ~28 people you could with the 60 second cooldown

The QoL is also a buff, because on several occasions you would just waste your 60 cooldown spell on Ian/Poly/Any other animal because they were on top of the target list.

The goal of this nerf was to push it into a more niche direction, rather than having it be your primary offensive ability it instead becomes a very useful emergency shutdown in case your disintegrate/fireball is on cooldown.

**Comment by trifornt-tinfornt** [comment]
18 cluwnes in a server that frequently has 60-80 players.
amazing spell.

We (myself and Hackole) told you that Cluwnes were already really underpowered and they didn't need further nerfs, and the spell was fine the way it was. Yet you literally ignored everything we said and are literally doing the exact same thing we said would be a BAD idea.

Tell me, why would someone use this over Disintegrate?

**Comment by TheOneAndOnlyCreeperJoe** [comment]
1/3 or 1/4 of the server being mentally retarded cluwnes incapable of anything is pretty big.
Especially if the other half is dead because you took disintegrate and used it on everyone you came across.

The point of the nerf is that players dont rely on it as their primary means of 'killing' people but rather as a tool used on people that actually pose a threat. If you want the ability to eliminate 2 people within short succession you take cluwne and disintegrate. You couldnt do this before unless you were using flesh to stone which is garbage without staff of animation.

**Comment by trifornt-tinfornt** [comment]
It's still, at this point, marginally inferior than Disintegrate. Usually you're going to have more than just Curse of the Cluwne on your hands. So why not just disintegrate one guy and kill him with another spell?

You kinda said yourself that the Cluwne Spell was MEANT to be a gimmick Ei Nath, that turned a lot of people into cluwnes for shits and giggles and general chaos without really killing anyone. Now it's actually supposed to be a legit spell, yet it has a recharge _that_ long?

**Comment by TheOneAndOnlyCreeperJoe** [comment]
"Kill him with another spell" There is no other spell, that's the thing. Fireball is slow and you dont always have time for it, FtS doesnt murder and the victims can be freed prematurely, Mutate also suffers from being a slow killer and Lightning bolt isn't ment for single target damage.

At first I thought that it would fit a gimmick, but then my vision on the spell changed. I think that the spell is great because it allows you to permanently turn someone into an useless cluwne from range and while stunned. It a great spell if you pair it with any other damage dealer. It makes up for the slowness of fireball and mutate and it is a great backup for when you cannot disintegrate the target you had in mind.

**Comment by blobbernaut** [comment]
I disagree with these cluwne nerfs, the only thing i'd do is make the spell not upgradeable. Cluwnes should still be able to do things, they have epilepsy already, there's just no point to nerfing them further, otherwise it's just oh i'm useless, and can't ghost, guess i'll just go play another game. 

You want the cluwne to still be able to have fun, so this nerf to them is bad.

**Comment by Nokington** [comment]
Yeah i'm not a fan of the spell nerfs.

**Comment by Lawligagger** [comment]
http://i.imgur.com/FZwn6NK.png thanks for watching the video kid

**Comment by Sarcalogo** [comment]
You should copy /vg/'s Cluwnes.

**Comment by calibraptor** [comment]
Axe the spell nerf, add in the cluwne one though.

**Comment by blobbernaut** [comment]
I'm not a fan of either nerf, the drop chance should be like 15% max

**Comment by Crystalwarrior** [comment]
I don't think this is a good PR, you should only have the "accidentaly drop thrown item" thing and axe off the spell nerf/etc.

**Comment by TheOneAndOnlyCreeperJoe** [comment]
They're not allowed to fight back most of the time anyway, unless they're attacking another valid (which most of the time is just the wiznerd), so the item dropping isn't actually that large of a deal.

**Comment by AliiTea** [comment]
This is a horrible idea.. worse than dildos.

**Comment by ghost** [comment]
Considering disintegrate goes from 60 seconds to 20 seconds, there would be no point taking it over disintegrate except for meme factor and there's no charge time.

Having it not be upgradable but still on a 60 second timer would be okay in my eyes.

**Comment by TheOneAndOnlyCreeperJoe** [comment]
Ehh sure I'll remove the cooldown increase.

**Comment by ghost** [comment]
I support this, cluwne should be a meme spell that curses someone to a fate worse than death. Not to be taken as a super serious spell. +1

**Comment by Lawligagger** [comment]
alrighty, gonna go ahead and merge this

**Comment by dreamcrab** [comment]
lets see how it happens in practice and then go from there, why not? 
+1 lets do it

**Comment by Nebyab** [comment]
ok

**Comment by TheOneAndOnlyCreeperJoe** [comment]
That's 2 heads supporting this, can this be merged?

**Comment by calibraptor** [comment]
Do it

File: code/_onclick/item_attack.dm [hunk 2]
```diff
@@ -79,6 +79,11 @@
 		return Clamp(src.w_class * 6, 10, 100) // Multiply the item's weight class by 6, then clamp the value between 10 and 100
 
 /obj/item/proc/attack(mob/living/M, mob/living/user, def_zone)
+	if(user.disabilities && CLUWNEMUT && prob(33))
```
Comment (Carbonhell) [inline code/_onclick/item_attack.dm]: It's user.disabilitues & CLUWNEMUT

File: code/modules/mob/living/carbon/carbon.dm [hunk 3]
```diff
@@ -262,7 +262,11 @@
 		unEquip(item)
 	if(src.client)
 		src.client.screen -= item
-
```
Comment (Carbonhell) [inline code/modules/mob/living/carbon/carbon.dm:265]: &

Comment (Carbonhell) [inline code/modules/mob/living/carbon/carbon.dm:265]: Also src. Is useless,remove

File: code/modules/reagents/reagent_containers/spray.dm [hunk 4]
```diff
@@ -18,6 +18,10 @@
 
 
 /obj/item/weapon/reagent_containers/spray/afterattack(atom/A as mob|obj, mob/user)
+	if(user.disabilities && CLUWNEMUT && prob(33))
```
Comment (Carbonhell) [inline code/modules/reagents/reagent_containers/spray.dm]: &

Comment (Carbonhell) [inline code/modules/reagents/reagent_containers/spray.dm]: Also should go after The if statement for tablets etc

Comment (Carbonhell) [inline code/modules/reagents/reagent_containers/spray.dm]: Tables

File: html/changelogs/Creeper Joe - CluwneDisabilities [hunk 5]
```diff
@@ -0,0 +1,10 @@
+author: Creeper Joe
```
Comment (Carbonhell) [inline html/changelogs/Creeper Joe - CluwneDisabilities]: Forgot .yml extension

**Comment by Carbonhell** [comment]
Code is all broken,fix it first

File: code/modules/mob/living/carbon/carbon.dm (hunk 3 above)
Comment (TheOneAndOnlyCreeperJoe) [inline code/modules/mob/living/carbon/carbon.dm:265]: Are you sure src is useless? Everything else in carbon/carbon uses src.

**Comment by TheOneAndOnlyCreeperJoe** [comment]
Carbon removing all the && that you indicated and replacing it with & breaks the code, causing runtime errors and causing you to always drop it/be unable to attack at all.

You are tearing me apart Carbon!

**Comment by Carbonhell** [comment]
Give me the runtimes perhaps so i can try to help nerd
I,m 100% sure about what i said

**Comment by TheOneAndOnlyCreeperJoe** [comment]
I fixed the shit you've mentioned.
Now hurry up already and merge this.

**Pull request merged**